### DISCUSSION TITLE
[WARNING] nonebot | Unknown adapter

### DISCUSSION THREAD

**Discussion by imcyx** [nonebot/discussions#4] [问答] [answered]
![image](https://user-images.githubusercontent.com/55662868/102690014-8f53bc80-423d-11eb-974a-fbe8145100cc.png)
想请教一下，按步骤配置过go-cqhttp和nonebot后为什么启动后一直错误，这是什么原因？

**Comment by bytemain** [comment, marked as answer]
https://github.com/nonebot/nonebot2/issues/110

忘了[register_adapter ](https://v2.nonebot.dev/guide/getting-started.html#%E5%8E%86%E5%8F%B2%E6%80%A7%E7%9A%84%E7%AC%AC%E4%B8%80%E6%AC%A1%E5%AF%B9%E8%AF%9D)
